docs/features/nullable-reference-types.md

```diff
@@ -267,7 +269,10 @@ A `class?` constraint is allowed, which, like class, requires the type argument
 
 An explicit `object` (or `System.Object`) constraint is allowed, which requires the type to be non-nullable when it is a reference type.
 However, an explicit `object?` constraint is not allowed.
-An unconstrained type parameter is essentially equivalent to one constrained by `object?`.
+An unconstrained (here it means - no type constraints, and no `class`, `struct`, or `unmanaged` constraints) type parameter is essentially
+equivalent to one constrained by `object?` when it is declared in a context where nullable annotations are enabled. If annotations are disabled,
```

Comment:
This contradicts [https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-08-09.md#unconstrained-generics](https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-08-09.md#unconstrained-generics) which has never been changed by the LDM.